### ISSUE TITLE
Compressed_vipc delay in streaming

### ISSUE THREAD

**Issue by Marcos64s** [commaai/openpilot#31460]
### Describe the bug

As compressed vipc is running an accumulated delay is detected on the total latency time, this is due to the network error incrementing as each frame is passed through. I suspect it has something to do with the first square of sensors, but im not sure. Any solutions to this? 
![Msg size_plot](https://github.com/commaai/openpilot/assets/78042816/746bb841-a58c-4df0-ac6a-f2d1244f037f)
![network lat_plot](https://github.com/commaai/openpilot/assets/78042816/022bf3ed-c73f-44a5-8d6d-5bb2e04d4504)
![pc_lat_plot](https://github.com/commaai/openpilot/assets/78042816/1fd282f2-23d6-4753-9fc3-fec0c1ad7791)
![process lat_plot](https://github.com/commaai/openpilot/assets/78042816/aed5e05d-0deb-4880-80d9-04297f78bc4b)
![Total time_plot](https://github.com/commaai/openpilot/assets/78042816/ca65c0ed-603e-4fd0-958b-e8f46cc0811c)

### Provide a route where the issue occurs

Not in a route

### openpilot version

0.9.6

**Comment by adeebshihadeh** [comment]
I'm not sure what your issue is exactly, and I can't reproduce any delayed frames. Can you post the code or a better description for the things you're plotting?

**Comment by Marcos64s** [comment]
The code is exactly the same as it is in the tools/camerastream/compressed_vipc.py (executed as the readme file). This times where gathered by setting the debug to TRUE set on the frame receiving device.

The issue represented in the graph is that the network delay should not increase over time, it should at least be constant or have small variations. Total time should not be an incrementing value, but have a range (based on the stream first measures) of around 100 ms, throughout the entire stream. 

I have a comma3 and the receiving device is a Jetson Orin (idk if this matters) inside the openpilot container, provided on the repo.

**Comment by adeebshihadeh** [comment]
I suspect this is related to your setup somehow.

**Comment by adeebshihadeh** [comment]
@Marcos64s still an issue?

**Comment by Marcos64s** [comment]
Yes, i m testing with new hardware. Soon i will leave an update.
